libvirt/cloudinit_def_test.go

```diff
@@ -202,6 +202,24 @@ func TestCreateCloudIsoViaPlugin(t *testing.T) {
 				  }`),
 				ExpectError: regexp.MustCompile("Error merging UserData with UserDataRaw: yaml: unmarshal errors"),
 			},
+
+			{
+				Config: fmt.Sprintf(`
+		      resource "libvirt_cloudinit" "test3" {
+			               name = "test3.iso"
+			               local_hostname = "tango12"
+			               pool =           "default"
+			    }`),
```

Comment:
* there's no need for `fmt.Sprintf`
* please format the config correctly (see 3fe13bb19661adfd488ff20d978fbac407e34835)

Reply:
OK THX ,  :sun#